conda.bzl

```diff
@@ -101,22 +109,32 @@ def _create_conda_build_file(rctx, executable):
 
 def _load_conda_impl(rctx):
     installer = _download_conda(rctx)
-    executable = _install_conda(rctx, installer)
-    _update_conda(rctx, executable)
-    _create_conda_build_file(rctx, executable)
+    conda_executable = _install_conda(rctx, installer)
+    if rctx.attr.install_mamba:
+        _install_mamba(rctx, conda_executable)
+    _update_conda(rctx, conda_executable)
+    _create_conda_build_file(rctx, conda_executable)
 
 load_conda_rule = repository_rule(
     _load_conda_impl,
     attrs = {
-        "conda_dir": attr.string(mandatory = True),
-        "version": attr.string(
+        "conda_dir": attr.string(mandatory=True),
+        "conda_version": attr.string(
             mandatory = True,
             doc = "Conda version to install",
         ),
+        "mamba_version": attr.string(
+            mandatory = True,
+            doc = "Mamba version to install"
+        ),
         "quiet": attr.bool(
             default = True,
             doc = "False if conda output should be shown",
         ),
+        "install_mamba": attr.bool(
+            default = True,
```

Comment:
Also I would not use `mamba` by default as discussed in https://github.com/spietras/rules_conda/issues/2#issuecomment-953234404, because people are mostly introduced only to `conda` first and `mamba` is just an addition and conscious decision to be made.